CHANGELOG

```diff
@@ -1,3 +1,55 @@
+1.5.0.rc1
+  # general
+  - You can now use LS_HOME/patterns directory to add generic patterns for those that may not be
+    associated with a particular plugin. Patterns in this dir will be loaded by default (#2225)
+  - We now check if the config file is correctly encoded. Otherwise we show a verbose error message
+    to convert the failing config file(s) to UTF-8 (#LOGSTASH-1103)
+  - Fixed bug in pipeline to gracefully teardown output workers when num workers > 1
+  - Fixed nologin path in release debian packages (#2283)
+  - Resolved issue where Logstash was crashing for users still using exclude_tags in their output
+    configuration (#2323)
+  - Updated and releases a new version of the logstash-event gem with latest changes in 1.5.0 RC1
+  - Windows: Significantly improved the initial user experience with Windows platform. Fixed many
+    issues in File input. Added support for using the plugin framework (installing, upgrading, removing)
+  - With the release of Kibana 4, we have removed the `bin/logstash web` command and any reference to
+    Kibana from Logstash. Kibana can be started very simply by running `bin/kibana` from the installed directory
+
+  # input
+  - Added new CouchDB input plugin to fetch data from CouchDB. Using the _changes API, data can be kept
```

Comment:
Might be explicit and follow with "This is intended to be a replacement for the Elasticsearch Couch River" or whatever